# Simple USB Camera Calibration Parameters
# Resolution: 320x240 (QVGA)
# Typical embedded/mobile camera
# Rujin Kim

image_width: 320
image_height: 240
camera_name: camera
camera_matrix:
  rows: 3
  cols: 3
  data: [525.0, 0.0, 320.0,
         0.0, 525.0, 240.0,
         0.0, 0.0, 1.0]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.0, 0.0, 0.0, 0.0, 0.0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.0, 0.0, 0.0,
         0.0, 1.0, 0.0,
         0.0, 0.0, 1.0]
projection_matrix:
  rows: 3
  cols: 4
  data: [525.0, 0.0, 320.0, 0.0,
         0.0, 525.0, 240.0, 0.0,
         0.0, 0.0, 525.0, 0.0]
